configure.ac

```diff
@@ -439,6 +439,11 @@ AC_LINK_IFELSE([AC_LANG_PROGRAM([[
 
 AC_SUBST(RKT_DEFINES_FOR_ENTER)
 
+# Check whether we can build TPM support code
+AC_CHECK_HEADER(trousers/tss.h, [TPM_TAGS=tpm], [], [AC_INCLUDES_DEFAULT])
```

Comment:
IIUC, this is just deciding to build with TPM based on whether the header is present; I'm wondering if we should instead make it a `./configure` flag. (This arguably collapses into my question about error handling; if that degrades safely in the no-tpm-actually-present-on-running-system case, then maybe we should just ALWAYS build with TPM?)